### ISSUE TITLE
To disable self-follow

### ISSUE THREAD

**Issue by smitbarmase** [DSC-VIT-BHOPAL/VITBOOK#5]
Owner's details are also showing up inside "_Search Vitians_" menu. 
following or unfollowing his/her own ID should not be allowed.
link to page:-https://vitbook.herokuapp.com/social/profile/

**Comment by Haugf** [comment]
I can pick up this issue! :-)

**Comment by Haugf** [comment]
What does invalid mean?

**Comment by smitbarmase** [comment]
Invalid means, this behavior of certain features is invalid. In this case, the search feature "Search Vitians" has some invalid functionality of following ourselves in the search result.

**Comment by Haugf** [comment]
Should your own profile still show up?

I have an idea that can filter the Search Vitians result on the frontend and avoid displaying the own users data

**Comment by smitbarmase** [comment]
No. We don't want to hide our own profile in search. We just need to hide the follow button on our profile in search.
